frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -236,6 +237,15 @@ if Device:isKobo() then
     }
 end
 
+if Device:isKindle() and PowerD:hasHallSensor() then
+    common_settings.hall_sensor = {
+        text = _("Hall Sensor"),
```

Comment:
You probably need a help_text:
`Detect when you are in a building or large room used for events involving a lot of people, and do nothing.`

(unlesss I didn't get what this does right :))